### DISCUSSION TITLE
在按照稀土掘金中课程进行配置，出现如下问题应该如何解决？

### DISCUSSION THREAD

**Discussion by MickeyCho630** [nshen/learn-neovim-lua#175] [Q&A]
[NvimTree]
Unknown option: view.hide_root_folder
Unknown option: view.mappings

see :help nvim-tree-opts for available configuration options

**Comment by MickeyCho630** [comment]
![Uploading Screenshot from 2024-09-11 17-20-36.png…]()

**Comment by MickeyCho630** [comment]
请问有没有大佬知道这个问题怎么解决呢？这个是我自己环境的问题，还是说是配置的问题？试过去掉这两个参数，去掉之后整个的配置就都不生效了，系统是Ubuntu 24.04 LTS

**Comment by nshen** [comment]
nvim-tree 组件更新修改过很多次参数,最新的配置参考 https://github.com/nshen/InsisVim/blob/main/lua/insis/plugins/nvim-tree.lua
